05/app.js

```diff
@@ -1,53 +1,70 @@
 const stats = {
-    paragraphs: {
-        'p1': 0,
-    },
-    links: {
-        '/dolor.html': 0,
-    }
+  paragraphs: {
+    p1: 0,
+  },
+  links: {
+    '/dolor.html': 0,
+  },
 };
 
 /* tutaj umieść swój kod */
 
+const pList = document.querySelectorAll('.text');
+
+const handleClick = function (e) {
+  e.preventDefault();
+  const href = e.target.getAttribute('href');
+  const id = e.target.dataset.id;
+  console.log(href);
+
+  if (href in stats.links) {
+    stats.links[href]++;
+  }
+
+  if (id in stats.paragraphs) {
+    stats.paragraphs[id]++;
+  }
```

Comment:
Co z kliknięciem w paragrafy czy linki, które na razie nie są uwzględnione na razie w statystyki?

Jak kliknę w drugi czy pierwszy to na liście z "wynikami" powinna pojawić się dodatkowa statystyka.